test/Metadata.cpp

```diff
@@ -36,14 +36,14 @@ bytes onlyMetadata(bytes const& _bytecode)
 	unsigned size = _bytecode.size();
 	if (size < 5)
 		return bytes{};
-	size_t metadataSize = (_bytecode[size - 2] << 8) + _bytecode[size - 1];
+	size_t metadataSize = (size_t(_bytecode[size - 2]) << 8) + _bytecode[size - 1];
```

Comment:
Why is this change needed? Is it because 8 is a signed constant?

Reply:
`_bytecode` is a vector of `uint8_t` and it needs cast. If there is no cast it does cast it to int and runs `<<` on int

Reply:
Oh, ok. `uint8_t` gets promoted to `int` rather than `unsigned`.